### PR TITLE
[4.1.0]Another set of links that was corrected that redirect the user to a page not found link

### PR DESCRIPTION
**Pull request by chamindi-a**

### CODE REVIEW

File: en/docs/reference/mi-security-reference/using_keystores.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Using Keystores
+    # Using Keystores
```
Comment (npamudika) [inline en/docs/reference/mi-security-reference/using_keystores.md:1]: We don't need the space here

File: en/docs/install-and-setup/setup/si-setup/configuring-cluster-coordination.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ should be configured for **all the nodes** as follows:
...
-| `             strategyConfig > datasource            `           | The shared datasource to be used in the cluster. The datasource specified must be properly configured in the `             deployment.yaml            ` file. For detailed instructions to configure a datasource, see [Configuring Datasources](_Configuring_Datasources_) .        | `             WSO2_CARBON_DB            `                                                      |
+| `             strategyConfig > datasource            `           | The shared datasource to be used in the cluster. The datasource specified must be properly configured in the `             deployment.yaml            ` file. For detailed instructions to configure a datasource, see [Configuring Datasources]({{base_path}}/install-and-setup/setup/si-setup/configuring-data-sources)    .        | `             WSO2_CARBON_DB            `                                                      |
```
Comment (npamudika) [inline en/docs/install-and-setup/setup/si-setup/configuring-cluster-coordination.md:42]: Do we need the space added after the closing curly brace?

Comment (chamindi-a) [reply en/docs/install-and-setup/setup/si-setup/configuring-cluster-coordination.md]: Fixed [d7baf75](https://github.com/wso2/docs-apim/pull/7325/commits/d7baf7539c3a8b3574cfc6f35b50f9285f28b3a2)

File: en/docs/reference/mi-security-reference/using_keystores.md (hunk 1 above)
Comment (chamindi-a) [reply en/docs/reference/mi-security-reference/using_keystores.md]: fixed [d7baf75](https://github.com/wso2/docs-apim/pull/7325/commits/d7baf7539c3a8b3574cfc6f35b50f9285f28b3a2)

File: en/docs/integrate/develop/customizations/creating-custom-task-scheduling.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Customizing Task Scheduling
+http://localhost:9000/soap/SimpleStockQuoteService# Customizing Task Scheduling
```
Comment (npamudika) [inline en/docs/integrate/develop/customizations/creating-custom-task-scheduling.md:1]: Let's remove the change in the title

Comment (chamindi-a) [reply en/docs/integrate/develop/customizations/creating-custom-task-scheduling.md]: Fixed [e8fadb7](https://github.com/wso2/docs-apim/pull/7325/commits/e8fadb7be21457a13e3e66ecfb9463152a8b0576)

**Review by npamudika — APPROVED** [review]

**Pull request merged**